sdk/dotnet/Appengine/Application.cs

```diff
@@ -0,0 +1,355 @@
+
+        /// <summary>
+        /// The [location](https://cloud.google.com/appengine/docs/locations)
+        /// to serve the app from.
+        /// </summary>
+        [Input("locationId")]
+        public Input<string>? LocationId { get; set; }
+
+        /// <summary>
+        /// Unique name of the app, usually `apps/{PROJECT_ID}`
+        /// </summary>
+        [Input("name")]
+        public Input<string>? Name { get; set; }
+
+        /// <summary>
+        /// The project ID to create the application under.
+        /// ~&gt;**NOTE**: GCP only accepts project ID, not project number. If you are using number,
+        /// you may get a "Permission denied" error.
+        /// </summary>
+        [Input("project")]
+        public Input<string>? Project { get; set; }
+
+        /// <summary>
+        /// The serving status of the app.
+        /// </summary>
+        [Input("servingStatus")]
+        public Input<string>? ServingStatus { get; set; }
+
+        [Input("urlDispatchRules")]
+        private InputList<Inputs.ApplicationUrlDispatchRulesGetArgs>? _urlDispatchRules;
+
+        /// <summary>
+        /// A list of dispatch rule blocks. Each block has a `domain`, `path`, and `service` field.
+        /// </summary>
+        public InputList<Inputs.ApplicationUrlDispatchRulesGetArgs> UrlDispatchRules
+        {
+            get => _urlDispatchRules ?? (_urlDispatchRules = new InputList<Inputs.ApplicationUrlDispatchRulesGetArgs>());
+            set => _urlDispatchRules = value;
+        }
+
+        public ApplicationState()
+        {
+        }
+    }
+
+    namespace Inputs
+    {
+
+    public sealed class ApplicationFeatureSettingsArgs : Pulumi.ResourceArgs
+    {
+        /// <summary>
+        /// Set to false to use the legacy health check instead of the readiness
+        /// and liveness checks.
+        /// </summary>
+        [Input("splitHealthChecks")]
+        public Input<bool>? SplitHealthChecks { get; set; }
+
+        public ApplicationFeatureSettingsArgs()
+        {
+        }
+    }
+
+    public sealed class ApplicationFeatureSettingsGetArgs : Pulumi.ResourceArgs
+    {
+        /// <summary>
+        /// Set to false to use the legacy health check instead of the readiness
+        /// and liveness checks.
+        /// </summary>
+        [Input("splitHealthChecks")]
+        public Input<bool>? SplitHealthChecks { get; set; }
+
+        public ApplicationFeatureSettingsGetArgs()
+        {
+        }
+    }
+
+    public sealed class ApplicationUrlDispatchRulesGetArgs : Pulumi.ResourceArgs
+    {
+        [Input("domain")]
+        public Input<string>? Domain { get; set; }
+
+        [Input("path")]
+        public Input<string>? Path { get; set; }
+
+        [Input("service")]
+        public Input<string>? Service { get; set; }
+
+        public ApplicationUrlDispatchRulesGetArgs()
+        {
+        }
+    }
+    }
+
+    namespace Outputs
+    {
+
+    [OutputType]
```

Comment:
indentation here makes me sad :'(

Reply:
Yeah, it's on my list of things to improve :/